Update requirements for pip, zip & ship (#168)
moderneinc/moderne-docs: docs/administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-an-agent-with-pypi-access-visualizations.md

* `/bin/sh` shell with
  * Python version 3.11 available under the `python3` command
  * Pip and virtualenv (venv) installed. Alias not required (called via `python3 -m pip/venv`)

> * The working directory us writable by the user that runs the agent jar, and has enough space to download the required packages.

Reviewer: ```suggestion
* The working directory is writable by the user that runs the agent JAR.
* The working directory has enough space to download the required packages.
```
